checklist.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: page
+permalink: /checklists/training/
+title: Checklist
+---
+## Scheduling the Event
+
+1.  Decide if it will be in person, online for one site, or online for several.
+1.  Talk through expectations with the host(s).
+    *   If it is in person, make sure the host knows they're covering travel costs.
```

Comment:
"...they're covering travel costs for trainers."

Be specific here because some hosts (depending on the context) may think that they are also supporting travel expenses for the participants.